### DISCUSSION TITLE
Push to F-Droid ?

### DISCUSSION THREAD

**Discussion by j0110** [Shabinder/SpotiFlyer#118] [General]
Thanks for your amazing work !
It is possible to make your app available in F-Droid repo ?

**Comment by Shabinder** [comment]
It is possible , but if I release on Fdroid , I wont be able to get any crashlytics/analytics which will help me to solve any crashes you encounter and moreover Fdroid version and Github releases cant be overwritten with each other due to different signatures. 

If people want to have this on FDroid , reply **+1** here, and I will consider it, if certain amount of users want the same.

**Reply by daes10** [reply]
A F-Droid repo would be nice.
I think much more people see your very nice work!

Thanks a lot

**Comment by mikhoul** [comment]
A big advantage for users to have it on Fdroid is thatit will update automatically with the Fdroid App.

I wanted to install the app for a friend that would not be able to update manually the app so that's why I was looking for a Fdroid version, IMO it would attract more casual users.

Regards :octocat:

**Comment by Shabinder** [comment]
SpotiFlyer is going to be on Fdroid :partying_face: 

Check Progress Here, 
https://gitlab.com/fdroid/fdroiddata/-/merge_requests/9019

**Comment by Shabinder** [comment]
Good News Guys :partying_face: :champagne: 

https://gitlab.com/fdroid/fdroiddata/-/merge_requests/9019 is Now **Merged**.
So Possible we will be seeing SpotiFlyer in FDroid in 2/3 Days, HopeFully.

**Reply by j0110** [reply]
Great ! That's good news. More people will be table to see your work !

**Comment by Shabinder** [comment]
Now get android app from fdroid 🍾 too: 
https://f-droid.org/packages/com.shabinder.spotiflyer/

**Reply by daes10** [reply]
Thanks!

**Comment by Satendra9984** [comment]
I am a beginner in android dev , I had learnt basic android app dev from google official course .How can I contribute to this project it already has lots of code ,please help me .
